ffmpeg_plugin/README.txt

```diff
@@ -1,24 +1,40 @@
 # svt-hevc ffmpeg plugin installation
 
+For Linux:
 1. Build and install SVT-HEVC 
 - git clone https://github.com/OpenVisualCloud/SVT-HEVC
 - cd SVT-HEVC/Build/linux
 - ./build.sh debug|release
 - cd debug|release
 - make install
+- export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
+- export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
+
+For Windows:
+0. Setup environment
+- Install Visual Studio 2017 or 2019
+- Install msys2-x86_64 to C:\msys64 following http://www.msys2.org/
+- Run "C:\msys64\mingw64.exe" to open a console
+- Install the required SW by "pacman -S make perl diffutils cmake yasm gcc pkgconfig git" in the console
+
+1. Build and install SVT-HEVC
+- git clone https://github.com/OpenVisualCloud/SVT-HEVC
+- cd SVT-HEVC/Build/windows
+- ./build.bat debug|release
+- cp SVT-HEVC/Bin/Debug|Release/SvtHevcEnc.dll c/svt-encoders/lib
+- export PKG_CONFIG_PATH=/c/svt-encoders/lib/pkgconfig:$PKG_CONFIG_PATH
+- pkg-config --print-errors SvtHevcEnc
```

Comment:
```suggestion
- pkg-config --print-errors SvtHevcEnc # To confirm pkg-config can find SvtHevcEnc
```

Since this isn't really necessary to building, better make a comment why